### ISSUE TITLE
Missing comma in Conv4d

### ISSUE THREAD

**Issue by Mathieu-Antonopoulos** [ZhengyuLiang24/Conv4d-PyTorch#3]
conv3d_layer = nn.Conv3d(in_channels=self.in_channels,
                                     out_channels=self.out_channels,
                                     kernel_size=self.kernel_size[1::],
                                     padding=self.padding[1::],
                                     dilation=self.dilation[1::],
                                     stride=self.stride[1::]
                                     bias=False)

There is a small syntax error after " stride=self.stride[1::]" line 75 in Conv4d.py. There should be a comma.

**Comment by Mathieu-Antonopoulos** [comment]
My bad, some other user had already commented about this issue
